doc/api/tls.md

```diff
@@ -1406,6 +1406,10 @@ changes:
     order as their private keys in `key`. If the intermediate certificates are
     not provided, the peer will not be able to validate the certificate, and the
     handshake will fail.
+  * `sigalgs` {string}` Colon-separated list of supported signature algorithms.
+    The list can contain digest algorithms (`SHA256`, `MD5` etc.), public key
+    algorithms (`RSA-PSS`, `ECDSA` etc.), combination of both (e.g
+    'RSA+SHA384') or TLS v1.3 scheme names (e.g. `rsa\_pss\_pss\_sha512`).
```

Comment:
nit: I don't think the `_` in a literal string needs back-slashing

This just gives examples, it isn't an exhaustive list, so users have to figure out, somehow, what the allowed values are. Is there an openssl man page we can link to?

Reply:
There is one, I can add a link to it: https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set1_sigalgs_list.html

Reply:
The man page is no more informative, so this will have to be good enough. Hopefully anyone using this knows why,.